controllers/testimonialsController.js

```diff
@@ -51,6 +51,29 @@ class TestimonialsController {
         msg: 'Testimonial was deleted successfully',
       });
     };
+
+    static async updateTestimonial(req, res){
+      let { id } = req.params
+
+      try {
+        let response = await Testimonials.update({...req.body}, { where : { id }})
```

Comment:
No hay sanitización o validación de campos enviados, así no es conveniente actualizar
[Más info](https://snyk.io/blog/sequelize-orm-npm-library-found-vulnerable-to-sql-injection-attacks/)